client/asset/btc/btc.go

```diff
@@ -1241,27 +1241,36 @@ func (btc *ExchangeWallet) FundingCoins(ids []dex.Bytes) (asset.Coins, error) {
 		if !notFound[pt] {
 			continue
 		}
-		txOut, err := btc.node.GetTxOut(txHash, rpcOP.Vout, true)
+
+		tx, err := btc.node.getTransaction(txHash)
```

Comment:
Just curious about the motivation for this change.  Same change is in `lockedSats`, but I don't recall why gettxout was an issue for this in spv, just not supported with btcwallet in spv?  But `swapConfirmations` is using gettxout, so I'm not clear on this.